README.md

```diff
@@ -7,3 +8,12 @@
 `npm run dev`
 
 All client side tamagotchi code is in Tamagotchi.tsx
+
+## Local Setup
+
+`brew install supabase/tap/supabase`
+
+`supabase start`
+
+To run migrations or reset database -- seed.sql and migrations will run
```

Comment:
**suggestion (documentation):** Clarify the migrations description

Consider rephrasing to: 'To reset the database and run migrations (including seed.sql):'

```suggestion
To reset the database and run migrations (including seed.sql):
```